.env.common-local

```diff
@@ -22,3 +22,8 @@ REACT_APP_UNSAFE_ERROR_SIMULATION=true
 #
 # Note: this variable is ignored when NODE_ENV is not `production`.
 PUBLIC_URL=/
+
+# Variables for Google Analytics
+REACT_APP_ANALYTICS_HOST=https://www.google-analytics.com
```

Comment:
this doesn't need to be specified again if it's already in the root `.env` I presume.